tests/test_call.py

```diff
@@ -5,6 +5,7 @@
 Boolean = jep.findClass('java.lang.Boolean')
 StringBuilder = jep.findClass('java.lang.StringBuilder')
 ArrayList = jep.findClass('java.util.ArrayList')
+LongStream = jep.findClass('java.util.stream.LongStream')
```

Comment:
We want to maintain Java 1.6 compatibility for as long as possible.  Can we test with something else?